domains/client/domain-operator/src/tests.rs

```diff
@@ -232,6 +232,14 @@ async fn setup_evm_test_nodes(
     evm_owner: impl Into<Option<Sr25519Keyring>>,
 ) -> (TempDir, MockConsensusNode, EvmDomainNode) {
     let evm_owner = evm_owner.into();
+    println!(
```

Comment:
`tracing::info!` would be better

Reply:
I tried that, but it wasn’t actually printing the test output, which is weird